hummingbot/connector/exchange/blocktane/blocktane_utils.py

```diff
@@ -26,29 +27,57 @@
                   is_connect_key=True)
 }
 
-TRADING_PAIR_SPLITTER = re.compile(r"^(\w+)(BTC|btc|ETH|eth|BRL|brl|PAX|pax)$")
+TRADING_PAIR_SPLITTER = re.compile(r"^(\w+)(BTC|btc|ETH|eth|BRL|brl|PAX|pax|USDT|usdt|PAXG|paxg|LETH|leth|EURS|eurs|LRC|lrc|BKT|bkt)$")
+MARKET_DATA = None
+INVERSE_MARKET_LOOKUP = None
+NAME_LOOKUP = None
 
 
 def split_trading_pair(trading_pair: str) -> Optional[Tuple[str, str]]:
-    try:
-        if ('/' in trading_pair):
-            m = trading_pair.split('/')
-            return m[0], m[1]
+    global MARKET_DATA
+    global INVERSE_MARKET_LOOKUP
+    global NAME_LOOKUP
+    if MARKET_DATA is None:
+        try:
+            markets = requests.get('https://trade.blocktane.io/api/v2/xt/public/markets').json()
+            MARKET_DATA = {market['id']: market for market in markets}
+            INVERSE_MARKET_LOOKUP = {(market['base_unit'].upper(), market['quote_unit'].upper()): market for market in markets}
+            NAME_LOOKUP = {market['name']: market for market in markets}
+        except Exception:
+            pass  # Will fall back to regex splitting
```

Comment:
I think this should be a separate private function (e.g. `_populate_lookups`), called from both `split_trading_pair` and `convert_to_exchange_trading_pair`